test/transaction.test.js

```diff
@@ -4,27 +4,26 @@ var expect = require('chai').expect;
 var testConnector = require('./connectors/test-sql-connector');
 
 var juggler = require('loopback-datasource-juggler');
-
+/* eslint-disable one-var */
 var db, Post;
 var Review;
-
+/* eslint-enable one-var */
```

Comment:
Is disabling one var needed here? I thought it's only needed in the context of a for loop: https://github.com/eslint/eslint/issues/5744

In addition, I'm surprised the top line isn't shortened to:

```
var db, Post, Review;
```